nelua/cdefs.lua

```diff
@@ -189,9 +192,90 @@ cdefs.target_info_code = [[
 #if defined(__ANDROID__)
   is_android = true;
 #endif
+#if defined(APPLE_IOS) || defined(IOS) || defined(__IPHONEOS__) || defined(TARGET_OS_SIMULATOR) || defined(__IOS__) || defined(__APPLE_IOS__) || defined(IOS_OBJ_C) || defined(TARGET_OS_IOS)
+  is_ios = true;
+#endif
 #if defined(__MACH__)
   is_mach = true;
 #endif
+#if defined(__HPUX__)
+  is_hpux = true;
+#endif
+#if defined(__SGI__)
+  is_irix = true;
+#endif
+#if defined(__TIZEN_H__) || defined(TIZEN_DEPRECATION) || defined(__TIZEN__)
+  is_tizen = true;
+#endif
+#if defined(__GAMEBOY__) || defined(__gba__) || defined(__GBA__)
```

Comment:
Isn't gameboy a different console than GBA? Maybe both could be separated.

Reply:
Yep it is, Done!